main.py

```diff
@@ -24,8 +25,14 @@ def imgcaption(sid, data):
 
 
 @sio.event
-def chat(sid, data):
-    msg = send_message(send_message(data["message"]))
+def chat(sid, data: dict):
+    if data.get("textgenwui") is not None:
+        msg = textgenwui.send_message(
+            data["message"],
+            "Assistant",  # CHANGE THIS TO WHATEVER THE CHARACTER'S NAME IS!!!!
```

Comment:
should be environment variable?

Reply:
too lazy

Reply:
can you please move this to a branch of this repo so i can work on it then

Reply:
hold on